procedures/candlepin/branch.md.erb

```diff
@@ -0,0 +1,21 @@
+# Branch Packaging
+
+- [ ] Clone https://github.com/theforeman/candlepin-packaging
+- [ ] Check out most recent branch `rpm/< latest_version >`
+- [ ] Create a new branch for the version being added
+  - `git checkout -b rpm/<%= release %>`
+- [ ] Update `package_manifest.yaml` with <%= release %>
+- [ ] Create project in copr `obal copr-project candlepin-staging-<%= release %>`
+- [ ] Push new branch to https://github.com/theforeman/candlepin-packaging
+
+# Update CI
+
+- [ ] Open a PR with the following updates:
+  - [ ] Add new vars file for the version being branched, [for example](https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/candlepin/4.4.groovy)
+  - [ ] Add <%= release %> to https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/yaml/includes/candlepin_versions.yaml.inc
+  - [ ] Add <%= release %> to https://github.com/theforeman/jenkins-jobs/blob/master/centos.org/jobs/candlepin-pipelines.yml
+- Create new GPG key for release
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('generate_gpg') %>
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('export_gpg_private') %> to back up
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('sign_gpg') %>
+  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('upload_gpg') %>
```

Comment:
```suggestion
  - [ ] VERSION=<%= release %> PROJECT=candlepin <%= rel_eng_script('upload_gpg') %>
- Publish the key
  - [ ] Use <%= rel_eng_script('upload_yum_gpg') %> to create [candlepin/<%= release %>/RPM-GPG-KEY-candlepin](https://yum.theforeman.org/candlepin/<%= release %>/RPM-GPG-KEY-candlepin) on yum.theforeman.org
```

This will require changes to `upload_yum_gpg_public_key.yaml` to support `candlepin`, but then we can just refer to that file in `katello-repos` and don't need to update it there (which is missing from this procedure too)

Reply:
https://github.com/theforeman/theforeman-rel-eng/pull/379